src/mxgraph-loader.ts

```diff
@@ -2,12 +2,24 @@ import factory, {type mxGraphExportObject} from 'mxgraph';
 
 export default initialize();
 
+// mxGraph globals
+declare global {
+  interface Window {
+    mxForceIncludes: boolean;
```

Comment:
This should always set to `false`. true is the to enable the development mode when developing mxGraph itself.
```suggestion
    mxForceIncludes: false;
```